x-pack/filebeat/module/cisco/ios/_meta/fields.yml

```diff
@@ -16,3 +21,39 @@
       forth). A facility can be a hardware device, a protocol, or a module of
       the system software. It denotes the source or the cause of the system
       message.
+
+  - name: multicast
+    type: group
+    description: >
+      Fields for XXXX
+    fields:
+    - name: group
+      type: integer
+      description: >
+        XXXX
```

Comment:
This seems to be missing some descriptions?

Reply:
yup, need to ask the original issue creator who identified the new log patterns what the fields are.